src/translator/logging.h

```diff
@@ -0,0 +1,35 @@
+#include "3rd_party/marian-dev/src/3rd_party/spdlog/spdlog.h"
+#include "common/logging.h"
+
+namespace marian {
+namespace bergamot {
+
+// RAII Wrap around logging, to clean up after the object on stack.
+class Logger {
+ public:
+  Logger() : marianLoggers_(createLoggers()) {
```

Comment:
Legit Wtf...

Then why do you have the marian name space?

Reply:
> Then why do you have the marian name space?

I understand offline that your concern here to be `bergamot::Logger`, instead of `marian::bergamot::Logger`, which doesn't really make all that difference to the problem we're setting out to solve. I would prefer not to have a `bergamot::` namespace with `Logger` only living with in. There's far too much convenience for keeping `marian::bergamot`. 


https://github.com/browsermt/bergamot-translator/pull/8#discussion_r562542754
![image](https://user-images.githubusercontent.com/727292/139581610-bb8875e0-e802-4689-bc3b-bb146494897a.png)

So far I only have a `marian::bergamot` namespace. Most my components are within this namespace. I understand I'm polluting something that `marian` may want to use in the future, but I suppose chances for this are slim. 

Marking this as resolved.